snuba/web/delete_query.py

```diff
@@ -44,11 +47,20 @@ def delete_from_storage(
     if not delete_settings.is_enabled:
         raise Exception(f"Deletes not enabled for {storage.get_storage_key().value}")
 
-    payload: dict[str, Any] = {}
+    # assert isinstance(columns, dict[str, list[Any]])
+    columns_is_expected_type = isinstance(columns, dict) and all(
+        [isinstance(k, str) and isinstance(v, list) for k, v in columns.items()]
+    )
+    if not columns_is_expected_type:
+        raise ValueError(
+            "expected input 'columns' to be of type dict[str, list[Any]] but it was not"
```

Comment:
This is kind of an odd response because the request is in json, so maybe some thing like "Invalid 'columns' object values: {columns.values()}' or something

I also think it could make sense to raise a `InvalidQueryException` instead